### PR TITLE
us-12512 - ux review comments addressed

### PR DESCRIPTION
**Pull request by SushreeM-h**
us-12512-updated

### CODE REVIEW

File: assets/i18n/en.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
   "YOUR_REF_NUMBER": "Your reference number",
   "POST_YOUR_SUPPORTING_DOCUMENTS": "Post your supporting documents",
   "WHAT_YOU_NEED_TO_DO_NOW": "What you need to do now",
-  "THE_INFO_YOU_HAVE_PROVIDED": "The information you have provided have been used to create your Child Benefit claim form",
+  "THE_INFO_YOU_HAVE_PROVIDED": "The information you have provided has been used to create your Child Benefit claim form",
```
Comment (gautamkumar-hmrc) [inline assets/i18n/en.json:59]: Why this change is not done in welsh

File: src/samples/ChildBenefitsClaim/ConfirmationPage.tsx [hunk 2]
```diff
@@ -93,17 +93,16 @@ const ConfirmationPage = ({ caseId, isUnAuth }) => {
...
             </div>
           </div>
           <h2 className='govuk-heading-m'> {t('WHAT_YOU_NEED_TO_DO_NOW')} </h2>
           {isUnAuth && isCaseRefRequired && <p className='govuk-body'>{t('PRINT_THIS_INFO')}</p>}
-          <p className='govuk-body'> {t('THE_INFO_YOU_HAVE_PROVIDED')} </p>
+          <p className='govuk-body'> {t('THE_INFO_YOU_HAVE_PROVIDED')}. </p>
           <ParsedHTML htmlString={documentList} />
-          <p className='govuk-body'> {t('HMRC_MIGHT_CALL_YOU')} </p>
           <p className='govuk-body'>
-            {' '}
```
Comment (gautamkumar-hmrc) [inline src/samples/ChildBenefitsClaim/ConfirmationPage.tsx:106]: You are making these changes in Auth also.

**Review by AkhilaHMRC — APPROVED** [review]
